components/layout.js

```diff
@@ -24,18 +24,21 @@ export default function Layout({ preview, children }) {
           href="/favicon/favicon-16x16.png"
         />
         {/* <link rel="manifest" href="/favicon/site.webmanifest" /> */}
-      <link
-        rel="mask-icon"
-        href="icon.svg"
-        color="#000000"
-      />
+        <link rel="mask-icon" href="icon.svg" color="#000000" />
         <link rel="shortcut icon" href="favicon.ico" />
         <meta name="msapplication-TileColor" content="#000000" />
         <meta
           name="msapplication-config"
           content="/favicon/browserconfig.xml"
         />
         <meta name="theme-color" content="#000" />
+
+        <link rel="preconnect" href="https://fonts.gstatic.com" />
+        <link
+          href="https://fonts.googleapis.com/css2?family=Poppins:wght@100;200;300;400;500;600;700;800;900&display=swap"
```

Comment:
just in case! please do check if we are using all the weights

Reply:
yeah, I just checked and we're only using medium and bold (500,700).

I can fix that right now 🚧

Reply:
nope... actually it's 400,500,600 and 700 👓